pinot-core/src/test/java/org/apache/pinot/queries/SelectionOnlyEarlyTerminationTest.java

```diff
@@ -62,7 +63,8 @@ public void testSelectOnlyQuery() {
       assertNull(brokerResponse.getResultTable());
       assertEquals(brokerResponse.getNumSegmentsProcessed(), numSegmentsPerServer * NUM_SERVERS);
       assertEquals(brokerResponse.getNumSegmentsMatched(), numThreadsPerServer * NUM_SERVERS);
```

Comment:
numSegmentsMatched should be between `NUM_SERVERS` to `numThreadsPerServer * NUM_SERVERS`